tests/ctests/test_netplan_parser.c

```diff
@@ -389,8 +389,8 @@ test_parse_utf8_characters(__unused void** state)
 {
     NetplanState *np_state = NULL;
     int fd;
-    int size;
-    int res;
+    size_t size;
+    long res;
```

Comment:
todo: `res = read(...)` returns a `ssize_t`, why are we defining it as a `long` up here?